src/services/news/model.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const newsSchema = new Schema({
+	schoolId: {type: Schema.Types.ObjectId, required: true},
+	title: {type: String, required: true},
+	content: {type: String, required: true},
+	url: {type: String, required: true},
+	createdAt: {type: Date, 'default': Date.now}
+},{
+	timestamps: true
+});
+
+const newsModel = mongoose.model('newsEntry', newsSchema);
```

Comment:
I think it's just auto-generated from feathers-generator, but `news` would be a better name for the db-model.